src/GlobalScreen/Scope/Layout/Builder/StandardPageBuilder.php

```diff
@@ -60,6 +65,16 @@ public function build(PagePartProvider $parts): Page
         $title = $parts->getTitle();
         $short_title = $parts->getShortTitle();
         $view_title = $parts->getViewTitle();
+        $toast_container = $this->ui->factory()->toast()->container();
+
+        foreach (
```

Comment:
This is just a minor thing but I would suggest to extract something into a variable for clarity.
Something like this:
```php
$toasts = (new ToastsOfNotifications($this->ui->factory(), new ilSetting('notifications'))
    ->transformNotificationsForToasts(
        (new ilNotificationOSDHandler())->getNotificationsForUser($this->user->getId())
    );
foreach ($toasts as $toast) {
    $toast_container = $toast_container->withAdditionalToast($toast);
}
```